app/helpers/friendships_helper.rb

```diff
@@ -0,0 +1,2 @@
+module FriendshipsHelper
```

Comment:
kindly delete this empty file.

Reply:
- Kindly cover the requirements from your last reviewer.